mappings/net/minecraft/client/render/GameRenderer.mapping

```diff
@@ -1,37 +1,135 @@
+	FIELD field_4270 zoom D
+	FIELD field_4271 zoomX D
+	FIELD field_4272 zoomY D
+	FIELD field_4273 anaglyphEnabled Z
 	FIELD field_4274 shader Lnet/minecraft/class_1212;
 	FIELD field_4275 SHADERS_LOCATIONS [Lnet/minecraft/class_1605;
 	FIELD field_4276 forcedShaderIndex I
 	FIELD field_4277 shadersEnabled Z
+	FIELD field_4278 frameCount I
+	FIELD field_4279 anaglyphFilter I
+		COMMENT Is set to 0 before enabling cyan filter and to 1 before enabling the red filter; otherwise unused.
 	FIELD field_4280 firstPersonRenderer Lnet/minecraft/class_1033;
 	FIELD field_4281 SHADER_COUNT I
 	FIELD field_4282 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_4283 RAIN_TEXTURE Lnet/minecraft/class_1605;
+	FIELD field_4284 SNOW_TEXTURE Lnet/minecraft/class_1605;
 	FIELD field_4285 client Lnet/minecraft/class_669;
 	FIELD field_4286 resourceManager Lnet/minecraft/class_1258;
 	FIELD field_4287 random Ljava/util/Random;
 	FIELD field_4288 viewDistance F
 	FIELD field_4289 mapRenderer Lnet/minecraft/class_682;
 	FIELD field_4290 ticks I
-	FIELD field_4300 lastSkyDarkness F
+	FIELD field_4291 targetedEntity Lnet/minecraft/class_1745;
+	FIELD field_4292 cursorXSmoother Lnet/minecraft/class_1708;
+	FIELD field_4293 cursorYSmoother Lnet/minecraft/class_1708;
+	FIELD field_4294 thirdPersonDistance F
+	FIELD field_4295 lastThirdPersonDistance F
+	FIELD field_4296 cursorDeltaX F
+		COMMENT The {@code cursorDeltaX} amd {@code cursorDeltaY} fields store the actual mouse movement without smoothing, and are only used when smooth camera is enabled.
+	FIELD field_4297 cursorDeltaY F
+	FIELD field_4298 smoothedCursorDeltaX F
+	FIELD field_4299 smoothedCursorDeltaY F
+	FIELD field_4300 lastTickDelta F
 	FIELD field_4301 movementFovMultiplier F
 	FIELD field_4302 lastMovementFovMultiplier F
+	FIELD field_4303 skyDarkness F
 	METHOD <init> (Lnet/minecraft/class_669;Lnet/minecraft/class_1258;)V
 		ARG 1 client
+		ARG 2 resourceManager
 	METHOD method_3352 areShadersSupported ()Z
 	METHOD method_3353 updateTargetedEntity (F)V
 		ARG 1 tickDelta
+	METHOD method_3354 updateFogColorBuffer (FFFF)Ljava/nio/FloatBuffer;
+		ARG 1 red
+		ARG 2 green
+		ARG 3 blue
+		ARG 4 alpha
+	METHOD method_3355 setupCamera (FI)V
+		ARG 1 tickDelta
+		ARG 2 anaglyphFilter
+	METHOD method_3356 render (FJ)V
+		ARG 1 tickDelta
+		ARG 2 nanoTime
+	METHOD method_3357 getFov (FZ)F
+		ARG 1 tickDelta
+		ARG 2 changingFov
+	METHOD method_3358 renderFog (IF)V
+		ARG 2 tickDelta
+	METHOD method_3359 renderWorld (IFJ)V
+		ARG 1 anaglyphFilter
+		ARG 2 tickDelta
+		ARG 3 limitTime
 	METHOD method_3360 onResized (II)V
+		ARG 1 width
+		ARG 2 height
+	METHOD method_3362 renderClouds (Lnet/minecraft/class_1036;FI)V
+		ARG 1 worldRenderer
+		ARG 2 tickDelta
+		ARG 3 anaglyphFilter
 	METHOD method_3363 loadShader (Lnet/minecraft/class_1605;)V
 		ARG 1 id
 	METHOD method_3364 onCameraEntitySet (Lnet/minecraft/class_1745;)V
 		ARG 1 entity
+	METHOD method_3365 getNightVisionStrength (Lnet/minecraft/class_1752;F)F
```

Comment:
`getNightVisionBrightness`?

Reply:
Strength makes more sense

Reply:
> Strength makes more sense, or it could be amplifier

no it doesnt you're saved because upstream